### PR TITLE
Feature/back urls parameters

### PR DESCRIPTION
**Pull request by ramiroarguelles**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [x] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [x] Requires translations.

### CODE REVIEW

**Comment by ramiroarguelles** [comment]
Estimados,
Actualice la documentación incorporando los campos que se retornan en la back-url.
Solamente incorpore en ES y EN. Quedaría pendiente PT.

Saludos,
Ramiro

**Comment by lamaralmp** [comment]
Aguardando validação de conteúdo (Key). Não adicionado a release 1.11.0.

File: guides/payments/web-payment-checkout/latest/advanced-integration.es.md [hunk 1]
```diff
@@ -233,6 +233,24 @@ Atributo |	Descripción
...
+
+Parámetro |	Descripción
+------------ 	|	--------
+`collection_id` | Retorna el ID del pago de Mercado Pago. |
+`collection_status` | Retorna el estado del pago. Ej.: `approved`para un pago aprobado o `pending` para un pago pendiente. |
+`external_reference` | Retorna el valor del campo `external_reference` que hayas enviado a la hora de crear la preferencia de pago. |
+`payment_type` | Retorna el tipo de pago. Ej.: `credit_card`para tarjetas de crédito o `ticket` para medios de pago en efectivo. |
+`merchant_order_id` | Retorna el ID de la orden de pago generada en Mercado Pago. |
+`preference_id` | Retorna el ID de la preferencia de pago a partir de la cuál se esta retornando. |
+`site_id` | Retorna el ID del país de la cuenta de Mercado Pago del vendedor. Ej.: MLA para Argentina, MLB para Brasil o MLM para México. |
```
[the remark this answers is not in the document]
Comment (ramiroarguelles) [reply guides/payments/web-payment-checkout/latest/advanced-integration.es.md:246]: Si se puede localizar por país genial :)

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 2]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
  `back_url`| **_pending._** Return URL for pending payment.
   | **_failure._** Return URL for canceled payment.
 
+Through the `back_url` the following parameters will return:
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:235]:
```suggestion
Through the `back_url` *the following parameters will return*:
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 3]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
...
+
+Parameter |	Description
+------------ 	|	--------
+`collection_id` | Returns the payment ID of Mercado Pago. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:239]:
```suggestion
`collection_id` | Payment ID of Mercado Pago. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 4]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
...
+`collection_status` | Returns the status of the payment. Ex: `approved` for an approved payment or` pending` for a pending payment. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:240]:
```suggestion
`collection_status` | Payment status. For example: `approved` for an approved payment or` pending` for a pending payment. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 5]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
...
+`external_reference` | Returns the value of the `external_reference` field that you sent when creating the payment preference. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:241]:
```suggestion
`external_reference` | Value of the `external_reference` field that you sent when creating the payment preference. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 6]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
...
+`payment_type` | Returns the type of payment. Ex: `credit_card` for credit cards or` ticket` for cash payment methods. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:242]:
```suggestion
`payment_type` | Payment type. For example: `credit_card` for credit cards or` ticket` for cash payment methods. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 7]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
...
+`merchant_order_id` | Returns the ID of the payment order created in Mercado Pago. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:243]:
```suggestion
`merchant_order_id` | Payment order ID created in Mercado Pago. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 8]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
...
+`preference_id` | Returns the ID of the preference from which you are returning. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:244]:
```suggestion
`preference_id` | Preference ID from which you are returning. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 9]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
...
+`site_id` | Returns the country ID of the seller's Mercado Pago account. Ex: MLA for Argentina, MLB for Brazil or MLM for Mexico. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:245]:
```suggestion
`site_id` | Returns the country ID of the seller's Mercado Pago account. For example: MLA for Argentina, MLB for Brazil or MLM for Mexico. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 10]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
...
+`processing_mode` | Returns the value `aggregator`. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:246]:
```suggestion
`processing_mode` | Value `aggregator`. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 11]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
...
+`merchant_account_id` | Returns with a value of `null`. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:247]:
```suggestion
`merchant_account_id` | Value `null`. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.en.md [hunk 12]
```diff
@@ -232,6 +232,22 @@ Attribute |	Description
...
+
+> Whether some of the parameters contain information will depend on whether the payer made the payment at the Mercado Pago Checkout and did not drop-off the flow before returning to your site through the `back_url` of **_failure._**.
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.en.md:249]:
```suggestion
> The information of the parameters will depend on the finalization of the payment at the Checkout of Mercado Pago and on the fact that the flow has not been abandoned before returning to your site through the ** back_url` of ** _ failure _ **.
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 13]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
 `auto_return` | Redireciona automaticamente para o seu site quando o pagamento é finalizado como aprovado. Os valores possíveis são `approved` e `all`.
  `back_url`| **_success._** URL de retorno perante pagamento aprovado.<br/><br/>**_pending._**  URL de retorno perante pagamento pendente.<br/><br/>**_failure._** URL de retorno perante pagamento rejeitado.
 
+A través de las `back_url` retornarán los siguientes parámetros:
+
+Parâmetro |	Descrição
+------------ 	|	--------
+`collection_id` | Retorna o ID do pagamento do Mercado Pago. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:237]:
```suggestion
`collection_id` | ID do pagamento do Mercado Pago. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 14]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
...
+`collection_status` | Retorna o estado do pagamento. Ex.: `approved` para um pagamento aprovado ou `pending` para um pagamento pendente. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:238]:
```suggestion
`collection_status` | Estado do pagamento. Ex.: `approved` para um pagamento aprovado ou `pending` para um pagamento pendente. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 15]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
...
+`external_reference` | Retorna o valor do campo `external_reference` que foi enviado no momento da criação da preferência de pagamento. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:239]:
```suggestion
`external_reference` | Valor do campo `external_reference` que foi enviado no momento da criação da preferência de pagamento. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 16]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
...
+`payment_type` | Retorna o tipo de pagamento. Ex.: `credit_card` para cartões de crédito ou `ticket` para meios de pagamento em dinheiro. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:240]:
```suggestion
`payment_type` | Tipo de pagamento. Ex.: `credit_card` para cartões de crédito ou `ticket` para meios de pagamento em dinheiro. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 17]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
...
+`merchant_order_id` | Retorna o ID da ordem de pagamento gerada no Mercado Pago. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:241]:
```suggestion
`merchant_order_id` | ID da ordem de pagamento gerada no Mercado Pago. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 18]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
...
+`preference_id` | Retorna o ID da preferência de pagamento a partir da qual se está retornando. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:242]:
```suggestion
`preference_id` | ID da preferência de pagamento a partir da qual se está retornando. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 19]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
...
+`site_id` | Retorna o ID do país da conta do Mercado Pago do vendedor. Ex.: MLA para Argentina, MLB para Brasil ou MLM para México. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:243]:
```suggestion
`site_id` | ID do país da conta do Mercado Pago do vendedor. Ex.: MLA para Argentina, MLB para Brasil ou MLM para México. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 20]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
...
+`processing_mode` | Retorna o valor `aggregator`. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:244]:
```suggestion
`processing_mode` | Valor `aggregator`. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 21]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
...
+`merchant_account_id` | Retorna com valor `null`. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:245]:
```suggestion
`merchant_account_id` | Valor `null`. |
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 22]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
...
+
+> Alguns dos parâmetros conterão informação apenas se o pagador realizou o pagamento no Checkout Mercado Pago e não abandonou o fluxo antes de retornar ao seu site através da `back_url` de **_failure._**
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:247]:
```suggestion
> Alguns dos parâmetros conterão informação apenas se o pagador realizou o pagamento no Checkout Mercado Pago e não abandonou o fluxo antes de retornar ao seu site através da `back_url` de **_failure_**.
```

File: guides/payments/web-payment-checkout/latest/advanced-integration.pt.md [hunk 23]
```diff
@@ -230,6 +230,21 @@ Atributo |	Descrição
...
+
+Parâmetro |	Descrição
+------------ 	|	--------
+`collection_id` | ID do pagamento do Mercado Pago. |
+`collection_status` | Estado do pagamento. Ex.: `approved` para um pagamento aprovado ou `pending` para um pagamento pendente. |
+`external_reference` | Valor do campo `external_reference` que foi enviado no momento da criação da preferência de pagamento. |
+`payment_type` | Tipo de pagamento. Ex.: `credit_card` para cartões de crédito ou `ticket` para meios de pagamento em dinheiro. |
+`merchant_order_id` | ID da ordem de pagamento gerada no Mercado Pago. |
+`preference_id` | ID da preferência de pagamento a partir da qual se está retornando. |
+`site_id` | ID do país da conta do Mercado Pago do vendedor. Ex.: MLA para Argentina, MLB para Brasil ou MLM para México. |
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/advanced-integration.pt.md:243]:
```suggestion
`site_id` | ID do país da conta do Mercado Pago do vendedor. Ex.: ----[mla]---- MLA para Argentina.------------ ----[mlb]---- MLB para Brasil.------------ ----[mlm]---- MLM para México.------------ ----[mpe]---- MPE para Perú.------------ ----[mlc]---- MLC para Chile.------------ ----[mco]---- MCO para Colombia.------------ ----[mlu]---- MLU para Uruguay.------------ |
```

**Pull request merged**